docs/tutorial/05-model-monitoring.ipynb

```diff
@@ -40,7 +40,8 @@
    "outputs": [],
```

Comment:
<p>In Jupyter, the output bar looks full and green, don't know why here its like this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>